ibis/backends/sql/rewrites.py

```diff
@@ -603,3 +603,19 @@ def lower_sample(_, **kwargs):
             "`Table.sample` with a random seed is unsupported"
         )
     return ops.Filter(_.parent, (ops.LessEqual(ops.RandomScalar(), _.fraction),))
+
+
+@replace(p.ArrayMap | p.ArrayFilter)
+def subtract_one_from_index_argument(_, **kwargs):
```

Comment:
For readability when looking at a rewrite list on a specific compiler, would you mind adding something to the rule name that better indicates this is for `ArrayMap`/`ArrayFilter` ops? Perhaps

```suggestion
def subtract_one_from_array_map_filter_index(_, **kwargs):
```